cs/Serializer/VowpalWabbitSerializer.cs

```diff
@@ -136,16 +176,37 @@ public VowpalWabbitExample Serialize(VowpalWabbit vw, TExample example, ILabel l
             }
             else
             {
-                result = new CacheEntry 
+                // TODO: catch exception and dispose!
```

Comment:
remove todo. (I wonder if anything else in native VW needs to be freed in exception state, but with VLD integrated the unit tests are probably enough)